planning/behavior_path_planner/src/utilities.cpp

```diff
@@ -1609,6 +1609,37 @@ std::shared_ptr<PathWithLaneId> generateCenterLinePath(
   return centerline_path;
 }
 
+// TODO(Azu) Some parts of is the same with generateCenterLinePath. Therefore it might be better if
+// we can refactor some of the code for better readability
+OccupancyGrid generateDrivableAreaForAllSharedLinestringLanelets(
```

Comment:
@zulfaqar-azmi-t4 
can you change this not to return Occupancy grid because this needs fuge copy so please pass with reference.

Reply:
or convert occupancy grid to line string would be better

Reply:
i'll try use Line Marker instead of Occupancy Grid.

Reply:
The line markers is now used instead of occupancy grid